noodles-bed/src/de.rs

```diff
@@ -0,0 +1,42 @@
+use std::io::BufRead;
+
+use crate::{Record, reader::Reader};
+
+pub fn from_reader<R>(reader: &mut Reader<R>) -> Vec<Record<3>>
+where
+    R: BufRead
+{
+    reader
+        .records::<3>()
+        // fails silently, bad design
+        // TODO: define correct error handling for both of these maps
```

Comment:
This shouldn't fail silently IMO and @mmalenic's: a single record failing should trigger an error.